### DISCUSSION TITLE
Datagrid Commands vs DataGridColumns with buttons?

### DISCUSSION THREAD

**Discussion by StephenOTT** [Megabit/Blazorise#2309] [Q&A] [answered]
Hi
Looking for some designed usage guidance on when to use Commands vs just custom buttons in columns?  is there anything special being provided?

**Comment by stsrki** [comment, marked as answer]
Commands just give out-of-the-box editing. With custom buttons in columns, you have all the power to make however you want. There is no rule here. Whatever works for your project is the way to go.

**Reply by StephenOTT** [reply]
Thanks
